.pylintrc

```diff
@@ -9,7 +9,7 @@
 
 # Add files or directories to the ignore list. They should be base names, not
 # paths.
-ignore=compat.py, __main__.py
+ignore=compat.py, __main__.py, zip.py
```

Comment:
good catch, I should remove it. Originally I added a link to the zip module to avoid the try catch while importing the zip module, but I found it was not working fine in windows. I will update it.